README.md

```diff
@@ -490,6 +490,45 @@ for example:
 1. `~/lrauv_ws/src/lrauv-application-2/bin/LRAUV`
 1. Start more `bin/LRAUV` as needed
 
+## Levels
+
+Some worlds support [levels](https://ignitionrobotics.org/api/gazebo/6.7/levels.html).
+Levels are turned off by default, which means that all heightmap tiles will
+be loaded at all times. When levels are enabled, only the tiles containing
+vehicles (performers) spawned with `WorldCommPlugin` with an
+[lrauv_init](https://github.com/osrf/lrauv/blob/main/lrauv_ignition_plugins/proto/lrauv_init.proto)
+message will be loaded.
+
+For example, loading without levels:
+
+`ign gazebo -v 4 portuguese_ledge.sdf`
+
+![portuguese_ledge_no_levels](https://user-images.githubusercontent.com/5751272/159629754-9206a9fe-eeed-4afc-a516-5c25a9fbdaf6.png)
+
+And with levels:
+
+`ign gazebo -v 4 portuguese_ledge.sdf --levels`
+
+![portuguese_ledge_no_levels](https://user-images.githubusercontent.com/5751272/159813230-809d1cad-b3f5-457a-af19-3752bec0710f.gif)
+
+#### Generating Portuguese Ledge world
+
+The generated file is committed to the repository. Whenever it is changed, a
+manual step is needed to regenerate it.
```

Comment:
@chapulina meta: why not generating the world upon package build? You may `add_custom_command` to interpolate `portuguese_ledge.sdf.em` during the build.

Reply:
I initially didn't want to force this to be rerun on every compilation because I think the world won't be changing often. It also adds a compile-time dependency to empy.

But I think it's a good idea for future-proofing, so done in 9a11eeed6615f55e69f671046de8d0f27ce0a0c7.